packages/consensus/src/validation/coinbase.cairo

```diff
@@ -30,11 +34,29 @@ pub fn validate_coinbase(
     let block_reward = compute_block_reward(block_height);
     assert(total_output_amount <= total_fees + block_reward, 'total output > block rwd + fees');
 
-    // TODO: validate BIP-141 segwit output
+    // validate BIP-141 witness field
+    if block_height >= BIP_141_BLOCK_HEIGHT {
+        let witness = tx.inputs[0].witness[0];
```

Comment:
There's `validate_coinbase_witness` that already does these checks

Reply:
Alright, seems this slipped through while rebasing my branch